src/BenchmarkDotNet.Toolchains.Roslyn/Builder.cs

```diff
@@ -12,7 +12,7 @@
 
 namespace BenchmarkDotNet.Toolchains.Roslyn
 {
-    internal class Builder : IBuilder
+    public class Builder : IBuilder
```

Comment:
@cdmihai, if you want to make something a part of public API, please mark it with `[PublicAPI]`.